app/src/main/java/de/danoeh/antennapod/dialog/FilterDialog.java

```diff
@@ -21,39 +28,103 @@ public FilterDialog(Context context, FeedItemFilter feedItemFilter) {
         this.filter = feedItemFilter;
     }
 
-    public void openDialog() {
-        final String[] items = context.getResources().getStringArray(R.array.episode_filter_options);
-        final String[] values = context.getResources().getStringArray(R.array.episode_filter_values);
-        final boolean[] checkedItems = new boolean[items.length];
+    public void openCustomDialog() {
```

Comment:
The method is still named `Custom`. Please just use the old name, so it does not require changes to other (unrelated) files